src/libraries/System.Private.Xml.Linq/tests/xNodeReader/IntegrityTest.cs

```diff
@@ -250,13 +249,15 @@ public void GetAttributeEmptyNameNamespace()
                 public void GetAttributeOrdinal()
                 {
                     XmlReader DataReader = ReloadSource();
+                    DataReader.Read();
```

Comment:
you should not be changing that I believe, we should check this code carefully as there might be some APIs depending on the fact that we used to return null and we should fix them to behave the same

Reply:
I naively checked all calls to GetAttribute(int) in the solution, and couldn't find any obvious special checks for return values of null. But I assume there's more to this? Is this something I can do?

Reply:
Should I just Assert.Throws<InvalidOperationException> here? The test before didn't really do a lot, since the calls immediately returned null due to not being in ReadState.Interactive..

Reply:
sounds fine in that case